benchmark/client/main.go

```diff
@@ -36,145 +37,160 @@ import (
 )
 
 var (
-	server            = flag.String("server", "", "The server address")
-	maxConcurrentRPCs = flag.Int("max_concurrent_rpcs", 1, "The max number of concurrent RPCs")
-	duration          = flag.Int("duration", math.MaxInt32, "The duration in seconds to run the benchmark client")
-	trace             = flag.Bool("trace", true, "Whether tracing is on")
-	rpcType           = flag.Int("rpc_type", 0,
+	port    = flag.String("port", "50051", "Localhost port to connect to.")
+	r       = flag.Int("r", 1, "The number of concurrent RPCs on each connection.")
```

Comment:
These guys are global variables, and as such, should have long names.  Rationale:

1. Used far away from where defined.
2. Shadowed by commonly-used locals.